pkg/workload/workload.go

```diff
@@ -687,3 +689,12 @@ func AdmissionChecksForWorkload(log logr.Logger, wl *kueue.Workload, admissionCh
 	}
 	return acNames
 }
+
+func EvictedEventReason(reason string) string {
```

Comment:
Exporting this function just for tests seems unnecessary. I would suggest to inline this function.

Actually, seeing "EvictedDueTo...` will make the tests more readable.

Reply:
Thanks. Done.